packages/ndla-notion/src/NotionBody.tsx

```diff
@@ -12,7 +12,10 @@ const NotionBody = styled.div`
     animation-duration: 500ms;
   }
   iframe {
-    height: 400px;
+    height: 619px;
+    ${mq.range({ until: breakpoints.tabletWide })} {
+      height: 300px;
+    }
```

Comment:
Ser dette fører til stor margin under visuelt element i noen tilfeller. Hvilket problem løser dette? Kunne det eventuelt forblitt 400px slik det var?

Reply:
Det er for mobilvisning, slik at det ikke blir så stor margin på mobil og tablet.

Reply:
Kan høyden på 619px endres til 400 igjen feks?

Reply:
Da må man scrolle iframen.